components/EditProfileForm/EditProfileForm.interface.ts

```diff
@@ -0,0 +1,11 @@
+export type EditableUserDetails = {
+  title: string;
+  description: string;
+  skills: string[];
+};
+
+export type EditProfileFormProps = {
+  currentTitle: string | undefined;
+  currentDescription: string | undefined;
+  currentSkills?: string[] | undefined;
```

Comment:
adding a ? is equivalent to adding a undefined fallback. 

```suggestion
  currentTitle?: string;
  currentDescription?: string;
  currentSkills?: string[];
```